src/PKSim.Core/Commands/SetRelativeExpressionFromNormalizedCommand.cs

```diff
@@ -0,0 +1,58 @@
+using System.Linq;
+using OSPSuite.Core.Domain;
+using PKSim.Core.Model;
+
+namespace PKSim.Core.Commands
+{
+   public class SetRelativeExpressionFromNormalizedCommand : PKSimMacroCommand
+   {
+      private readonly double _newNormalizedValue;
+      private readonly IParameter _normalizedParameter;
+
+      public SetRelativeExpressionFromNormalizedCommand(IParameter normalizedParameter, double normalizedValue)
+      {
+         _normalizedParameter = normalizedParameter;
```

Comment:
Did you check how it looks in the history? I think Description, types etc are all missing.
You may be able to bypass that by setting one of the comand in the macro commadn to visible = false but this needs to be tested